apollo-federation/src/sources/connect/json_selection/README.md

```diff
@@ -381,9 +380,6 @@ In addition to renaming, `Alias` can provide names to otherwise anonymous
 structures, such as those selected by `PathSelection`, `NamedGroupSelection`, or
 `StarSelection` syntax.
 
-Because we always want to generate GraphQL-safe output properties, an `Alias`
-must be a valid GraphQL identifier, rather than a quoted string.
-
```

Comment:
Right, see https://github.com/apollographql/router/pull/5987/commits/0314539954d4aa5bbf1af61c0e03478bff0804ed. I included that change in this PR because I believed it would affect how `Alias` tracks its location (by wrapping a `Key`). I still think it's a worthwhile change, though I suppose we could have used `WithRange<String>` to achieve a similar effect, without switching to `Key`.